tfe/resource_tfe_team.go

```diff
@@ -36,6 +36,7 @@ func resourceTFETeam() *schema.Resource {
 				Type:     schema.TypeList,
 				MaxItems: 1,
 				Optional: true,
+				Computed: true,
```

Comment:
@bendrucker 🙇  Thought you'd be interested in this, since we were discussing Computed earlier...

Turns out this should be computed as it's _sometimes_ an API-computed set of values, which triggers an updating of state on creation with those fetched values. It can continue to be optional, so you can still set them _if_ you want to. 

This is a broader use-case than what is currently documented for Computed:

> Computed is often used to represent values that are not user configurable or can not be known at time of terraform plan or apply, such as date of creation or a service specific UUID.

It's still _technically_ true (we don't know what the value will be since it may or may not be returned) but it's weird because other times we _can_ know what the values will be (because we set them). I plan on getting that explained a little better there, since I think that's pretty confusing. 

Visibility will remain _not_ computed as that attribute is _always_ returned and has a true default case that doesn't suffer from the same thing as we're talking about here.